src/server/serve.rs

```diff
@@ -116,17 +116,21 @@ impl InnerService {
     /// 1. Remove leading slash.
     /// 2. Strip path prefix if defined
     /// 3. URI percent decode.
-    /// 4. Concatenate base path and requested path.
+    /// 4. If on windows, switch slashes
+    /// 5. Concatenate base path and requested path.
     fn file_path_from_path(&self, path: &str) -> Result<Option<PathBuf>, Utf8Error> {
         let decoded = percent_decode(path[1..].as_bytes())
             .decode_utf8()?
             .into_owned();
-
-        let stripped_path = match self.strip_path_prefix(&decoded) {
+        let slashes_switched = if cfg!(windows){
+            decoded.replace("/", "\\")
+        } else {
+            decoded
```

Comment:
By move line 124 `into_owned()` to here, we can reduce one unnecessary allocation.
```suggestion
            decoded.into_owned()
```

Reply:
When I try to build with this change, I get "method not found in `std::string::String`".  Could easily be me missing something (still learning rust), but if so I don't know what.

Reply:
Oh. I mean we can also remove line 124 😂

```diff
        let decoded = percent_decode(path[1..].as_bytes())
-            .decode_utf8()?
-            .to_owned();
+            .decode_utf8()?;
```

Reply:
Okay, I went ahead and made both your suggested changes (then remembered to rerun `cargo fmt` >.>).